site/en/docs/extensions/mv3/manifest/activeTab/index.md

```diff
@@ -81,7 +81,7 @@ navigated or is closed.
 
 While the `activeTab` permission is enabled for a tab, an extension can:
 
-- Call [`tabs.executeScript`][5] or [`tabs.insertCSS`][6] on that tab.
+- Call [`tabs.insertCSS`][6] on that tab.
```

Comment:
Should this be `chrome.scripting.insertCSS`?

Reply:
Also `chrome.scripting.removeCSS`